frontend/src/app/patients/AddPatient.tsx

```diff
@@ -289,11 +289,12 @@ const AddPatient = () => {
 
   const getSaveButtons = (
     formChanged: boolean,
-    onSave: (startTest?: boolean) => void
+    onSave: (startTest?: boolean) => void,
+    location: string
   ) => (
     <>
       <Button
-        id="edit-patient-save-lower"
+        id={`edit-patient-save-and-start-${location}`}
```

Comment:
How do these changes relate to this ticket?

Reply:
This change was specific to another dequeue issue I noticed while inspecting the page. This button was duplicated multiple places and had the same id.

Reply:
Gotcha! I didn't connect that to your PR comments above 🤦🏼‍♀️